### ISSUE TITLE
Full screen underutilized...

### ISSUE THREAD

**Issue by iherman** [w3c/presentation-viewer#13]
This may be difficult to achieve, so just pinging this... but the full screen possibility for the browser is pretty much underutilized. See the attached screen shot...

Again, it may be too complex to do something about it; if so, this is a minor issue that can be ignored...

![Screen Shot 2020-05-05 at 08 55 57](https://user-images.githubusercontent.com/520723/81041569-79f87080-8eae-11ea-839b-153adf79f94d.png)

**Comment by r12a** [comment]
Sort of connected, i'd like to have a link that moves the sync video button to the top of the screen.  On my iPad, in particular, you have to manually adust the scroll to fit the video and slides on the screen, but  a lot of space is wasted by the large SLIDES & VIDEO title and keyboard shortcut prompt when linking to that section of the page.

**Comment by bert-github** [comment]
Re Ivan's comment: I didn't want to make the text (much) bigger than what you have configured in your browser. You may have a very big screen, but you are probably sitting at a distance that makes that text comfortable to read. If you want it bigger, you can zoom in. But I don't want to assume that people want bigger text because they have a bigger screen.

Re r12a's comment: There is a link in the first section called "slides" that puts the slides at the top of the screen. I can easily make it link to the sync button instead. Would that help?

**Comment by r12a** [comment]
> Re r12a's comment: There is a link in the first section called "slides" that puts the slides at the top of the screen. I can easily make it link to the sync button instead. Would that help?

Ah, that wasn't very noticeable, especially as it's embedded in some text that didn't attract my attention because it didn't look important. ;-)

What would be nice is to have something right there, next to the slides/video that you can click on.  There's a downwards pointing arrow that you can use to jumpt to the next section (but which btw seems to be pretty redundant for the SLIDES & VIDEO section).  I suspect that something similar, perhaps pointing upwards to a horizontal line just above (ie. something like 

![2B71](https://user-images.githubusercontent.com/4839211/81088498-eac17c00-8ef2-11ea-957c-0364c93623d9.png)

and positioned alongside the other arrows alongside the Synch video... control might work(?)  Just an idea.

**Comment by iherman** [comment]
> Re Ivan's comment: I didn't want to make the text (much) bigger than what you have configured in your browser. You may have a very big screen, but you are probably sitting at a distance that makes that text comfortable to read. If you want it bigger, you can zoom in. But I don't want to assume that people want bigger text because they have a bigger screen.

I am not sure what you say: this is not a matter of my configuration. 

If I have a shower presentation (or reveal, for that matter), and I put my browser into full screen, the slide will automatically fill the whole screen. (I am not sure how the underlying script does it, to be honest.) What I expected to see is something similar: in some way or other, the full screen should include the slides and the video side-by-side, making use of the screen real estate as much as possible.

Again, achieving this may be too complicated...

**Comment by bert-github** [comment]
I think I understand now what you expected. You want a mode in which there is nothing else than the slides and the video, no scrollbar, just a button to get back to the web page. Richard's button to align the slides to the top of the window would not be needed, the view wouldn't scroll anyway.

But what we have currently is a typical web page, with a header, a footer, links to other pages and a scrollbar. Even when the video is playing, you're still in ‘web browsing mode’ and you can scroll the page or click a link at any time. The button that is there doesn't hide the web page, it just hides some of the slides and transcripts.

I think we should test such a ‘maximize’ function next time.

As for techniques: Shower, Reveal.js and b6+ hide all slides except one and then scale the document with a CSS style rule 'transform: scale(s)', where s is the window width divided by the slide width. Some other tools (S5, Slidy, b5) increase the font size. The former gives the most faithful rendering of the slide as you wrote it (modulo anti-aliasing), but may cause black bands if the window doesn't have the same aspect ratio as the original slide (letterbox effect). The latter uses the full window, but lines may break differently.

**Comment by iherman** [comment]
Yes @bert-github, you give an accurate description of what I was looking for. And I realize it isn't obvious to do it, ie, not before the AC meeting. Let it be part of the some-day-pile...
